lib/ECSManager.js

```diff
@@ -82,6 +82,7 @@ class ECSManager {
 		taskDefinitionString = Util.substitute(taskDefinitionString, "cpu", Config.str("cpu"));
 		taskDefinitionString = Util.substitute(taskDefinitionString, "dockerHubSecretArn", Config.str("dockerHubSecretArn"));
 		taskDefinitionString = Util.substitute(taskDefinitionString, "image", Config.str("image"));
+		taskDefinitionString = Util.substitute(taskDefinitionString, "launchType", Config.str("launchType"));
```

Comment:
I believe the second parameter needs to be set to "FARGATE" - if we do not default this, it can cause backwards compatibility issues.

Reply:
I added `launchType` to our secrets manager defaults to prevent that. The launchType is set to FARGATE there.